plugins/ksp/src/org/jetbrains/kotlin/ksp/processing/impl/ResolverImpl.kt

```diff
@@ -325,11 +295,10 @@ class ResolverImpl(
     }
 
     // TODO: local scope
```

Comment:
Looks like we can remove this TODO now since it is no longer returning null.

Reply:
Done.